ex1.py

```diff
@@ -0,0 +1,22 @@
+FUNCTION_STOP_CHAR = "q"
+
+def calculate_average():
+    counter = 0
+    sum = 0
+    num = input('enter num if you want to stop enter {} '.format(FUNCTION_STOP_CHAR))
```

Comment:
f'enter num if you want to stop enter {FUNCTION_STOP_CHAR} '

this way its more clear